tomviz/MainWindow.cxx

```diff
@@ -382,6 +382,21 @@ QString MainWindow::readScript(const QString &scriptName)
     QByteArray array = file.readAll();
     return QString(array);
   }
+// On OSX the above doesn't work in a build tree.  It is fine
+// for superbuilds, but the following is needed in the build tree
+// since the executable is three levels down in bin/tomviz.app/Contents/MacOS/
+#if __APPLE__
+  else
+  {
+  path = QApplication::applicationDirPath() + "/../../../../share/tomviz/scripts/";
```

Comment:
It prints .../tomviz_build/bin/tomviz.app/Contents/MacOS

I can "fix" my problem by changing the directory in line 377 from "/../share/tomviz/scripts/" to "/../../../../share/tomviz/scripts/".
Maybe Tomviz never run the code between line 388 to line 400?